cmd/snap/cmd_info.go

```diff
@@ -163,27 +163,68 @@ func coalesce(snaps ...*client.Snap) *client.Snap {
 	return nil
 }
 
-// formatDescr formats a given string (typically a snap description)
+func rTrimRightSpace(text []rune) []rune {
+	j := len(text)
```

Comment:
Why `rTrimRightSpace` and not just `trimRightSpace`?

Reply:
**r**unes

Reply:
That's the type, which is impossible to get wrong since the compiler will complain.
So I agree with Zyga, trimRightSpace seems much cleaner.